src/containers/HomePage.spec.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { fromJS } from 'immutable';
+// import nock from 'nock';
+
+import HomePage from './HomePage';
+import configureStore from '../store/configureStore';
+import { withStore } from '../utils/testHelpers';
+
+describe('<HomePage />', () => {
+  let store;
+  let subject;
+
+  beforeEach(() => {
+    store = configureStore(fromJS({
+      session: {
+        authenticated: true,
+        checked: true,
+        invalid: false,
+        user: {
+          id: 300,
+          username: 'Peter'
+        }
+      }
+    }));
+    const location = {
+      pathname: '/',
+      search: '/',
+      hash: '',
+      key: '9kzkus'
+    };
+    subject = mount(withStore(<HomePage location={location} />, store, location));
+  });
+
+  it('should render a div', () => {
```

Comment:
`a div`? There are many divs, maybe you should specify that you're looking for an specific root div tag.